configuration_go/schemas/thanos/tracing/client/client.go

```diff
@@ -17,3 +19,12 @@ type TracingConfig struct {
 	Type   TracingProvider `yaml:"type"`
 	Config interface{}     `yaml:"config"`
 }
+
+func (c TracingConfig) String() string {
+	// We use "gopkg.in/yaml.v2" instead of "github.com/ghodss/yaml" for correct formatting of this config.
```

Comment:
This comment can be put on the function as a goDoc comment, so that user can see it?

Reply:
done